components/Lead.vue

```diff
@@ -2,19 +2,33 @@
   <div>
     <div class="lead">
       <h2 class="lead__title">{{ content.title }}</h2>
-      <p class="lead__dates">{{ content.dates }}</p>
+      <CommonLink :href="calendarUrl">
+        <span class="lead__dates">{{ content.dates }}</span>
+        <AddToCalIcon />
+      </CommonLink>
     </div>
   </div>
 </template>
 
 <script>
+import AddToCalIcon from '~/assets/svg/add_to_cal.svg?inline'
+
 export default {
+  components: {
+    AddToCalIcon,
+  },
   props: {
     content: {
       type: Object,
       required: true,
     },
   },
+  data() {
+    return {
+      calendarUrl:
+        'http://www.google.com/calendar/event?action=TEMPLATE&dates=20210608%2F20210610&text=Global%20Maintainer%20Event&&details=A%20virtual%20gathering%20for%20the%20maintainers%20that%20make%20open%20source%20possible',
```

Comment:
This will be a bit hard to change in the future (I'm not certain that it will change a lot though). Do you think is worth to compute this based on some data? I assume that we have this in the page content, so maybe we can compute it from it. What do you think?

Reply:
Great idea! I'll update the PR with your suggestion in a sec, thx! 👏🏻

Reply:
It has taken me more than expected, could you review [the last commit](https://github.com/codegram/globalmaintainersummit.github.com/pull/22/commits/cf3cc1b76b793bee6aaa4cff32e57cd1d1c580c0) please? 😄